src/blocks/common.blocks/content/_view/content_view_showcase.styl

```diff
@@ -0,0 +1,4 @@
+.content_view_showcase .iframe
+{
+    position: absolute; // fix for iframe 100%-height in webkit
```

Comment:
@tavriaforever Because `iframe` is `display:block` by default. `position: absolute` is hack for webkit, where content does not stretched in height
![2015-04-17_1118](https://cloud.githubusercontent.com/assets/1333220/7198356/310fe5c4-e4f4-11e4-9b40-ba3783452e66.png)